### ISSUE TITLE
Error message after selecting option 1 in main menu

### ISSUE THREAD

**Issue by Feeclichette** [haimbar/RCMap#3]
Hi,

I wrote to you a few weeks ago regarding problems I was having with entering my data into RCMap. I made all the corrections you suggested, and reviewed all my files to ensure there were no data entry errors. However, when I select my data folder from option 1 of the main menu, the following error message appears (see attached screenshot). Also, it seems that I don't have access to the other options in the main menu if my data folder is not supported correctly by the software from option 1. So I am resending my corrected files to you, hoping that I can remedy these obstacles, and thanking you once again in advance for your time and help.

<img width="593" alt="Capture d’écran 2023-09-26 135149" src="https://github.com/haimbar/RCMap/assets/143531426/4b87c104-c067-4113-ba6a-c7e5894bf138">

**Comment by haimbar** [comment]
The problem is caused by two invalid sorted card numbers. The Statements.csv file contains statements 1-60, but in SortedCards.csv there are statement IDs 66 and 74. Once I removed them, the project loaded correctly.

**Comment by Feeclichette** [comment]
Thank you for your prompt response. I know about the wrong statement numbers in the old files, but I edited and corrected my files (commas instead of semicolons, checking for wrong numbers, etc.). My current files are attached. With the correct files and information, however, I continue to receive an error message when I select my folder with option 1 from the main menu. I don't understand where the bug is (the exact error message is attached). I really hope I can analyze my data and complete my project with RCMap because my department can't afford some of this other expensive software. Thanks again in advance.
[Demographics.csv](https://github.com/haimbar/RCMap/files/12793526/Demographics.csv)
[Ratings.csv](https://github.com/haimbar/RCMap/files/12793527/Ratings.csv)
[SortedCards.csv](https://github.com/haimbar/RCMap/files/12793528/SortedCards.csv)
[Statements.csv](https://github.com/haimbar/RCMap/files/12793529/Statements.csv)
<img width="593" alt="Capture d’écran 2023-09-26 135149" src="https://github.com/haimbar/RCMap/assets/143531426/0a3b9668-c596-4267-abd2-df705fbbda4f">

**Comment by Feeclichette** [comment]
Thank you for your prompt response. I know about the wrong statement
numbers in the old files, but I edited and corrected my files (commas
instead of semicolons, checking for wrong numbers, etc.). My current files
are attached. With the correct files and information, however, I continue
to receive an error message when I select my folder with option 1 from the
main menu. I don't understand where the bug is (the exact error message is
attached). I really hope I can analyze my data and complete my project with
RCMap because my department can't afford some of this other expensive
software. Thanks again in advance.

**Comment by haimbar** [comment]
I opened the latest file you attached, and in row number 127 I still see the invalid entry 66, and in row 100 I see the invalid entry 74. These two entries cause the error. If I just delete these two values, the project loads successfully.

**Comment by Feeclichette** [comment]
My mistake, really sorry, I attached the old one. Here is the actual one,
without entrees 66 and 74 in rows 127 and 100, but with the same mistake
error (Erreur dans issues %+% yellow("Sorter ", i, " did not sort card(s)
", +cardNames[which(rsum ==  :
  length(lhs) == length(rhs) || length(lhs) == 1 || length(rhs) ==  ....
n'est pas TRUE)

**Comment by haimbar** [comment]
I am able to successfully load the project when these entries are removed. I did not get an attachment with your latest posting. I think that at this point, since I can't reproduce your problem, it would be best if you contact me at my UConn email, and we can arrange a zoom call to see why you're having trouble loading the project.

**Comment by Feeclichette** [comment]
Ok, thanks again. It's strange that you didn't receive my last attachment,
I can see it in my sent messages. :(
I think I was able to make some progress, because I reentered my files and
obtained some other error messages, including that many of my sorters (10)
put more than a third of the cards in one pile... I'm afraid that I'll have
to remove these 10 and keep a sample of only 17 sorters. Unfortunately, it
was not in my starting instructions to participants to put a maximum X of
cards in each pile. Does that mean I can't analyze the data of that 10
participants with the software?
Finally, I don't understand the other error messages (see printscreen in
attachment) : "Erreur dans dimnames(x) <- dn :
  la longueur de 'dimnames' [1] n'est pas égale à l'étendue du tableau
De plus : Messages d'avis :
1: Dans initCMap(dataDir) : NAs introduits lors de la conversion automatique
2: Dans getAdjMatrices(cardDat, cardNames) :
  NAs introduits lors de la conversion automatique"

I resend you all my files (supposed to be the good ones!) and the last
printscreen.

I really appreciate you offer the Zoom and your time to help me with my
project. I'm not available on thursdays, but I can plan a meeting on any
other week day between 9 and 4, at your convenience.

Thank you very much again and hope I will be able to obtain some results!

**Comment by Feeclichette** [comment]
Hi!
I found another data problem in the SortedCards.csv file. In Line 104 you have a space in the second column. I changed it to a period, it loads fine.


16,Activité, sport et physique,5,7,11,12,15,19,27,30,31,35,37,40,43,46,50,53,,,,,,,,,,,,,,,,,,,,,,,,,,

Haim
*Message sent from a system outside of UConn.*

Ok, thanks again. It's strange that you didn't receive my last attachment, I can see it in my sent messages. :(
I think I was able to make some progress, because I reentered my files and obtained some other error messages, including that many of my sorters (10) put more than a third of the cards in one pile... I'm afraid that I'll have to remove these 10 and keep a sample of only 17 sorters. Unfortunately, it was not in my starting instructions to participants to put a maximum X of cards in each pile. Does that mean I can't analyze the data of that 10 participants with the software?
Finally, I don't understand the other error messages (see printscreen in attachment) : "Erreur dans dimnames(x) <- dn :
  la longueur de 'dimnames' [1] n'est pas égale à l'étendue du tableau
De plus : Messages d'avis :
1: Dans initCMap(dataDir) : NAs introduits lors de la conversion automatique
2: Dans getAdjMatrices(cardDat, cardNames) :
  NAs introduits lors de la conversion automatique"

I resend you all my files (supposed to be the good ones!) and the last printscreen.

I really appreciate you offer the Zoom and your time to help me with my project. I'm not available on thursdays, but I can plan a meeting on any other week day between 9 and 4, at your convenience.

Thank you very much again and hope I will be able to obtain some results!


Le mar. 3 oct. 2023 à 10:49, haimbar ***@***.******@***.***>> a écrit :

I am able to successfully load the project when these entries are removed. I did not get an attachment with your latest posting. I think that at this point, since I can't reproduce your problem, it would be best if you contact me at my UConn email, and we can arrange a zoom call to see why you're having trouble loading the project.
